noetic/distribution.yaml

```
@@ -10470,6 +10470,19 @@ repositories:
       url: https://github.com/tilk/rtcm_msgs.git
       version: master
     status: maintained
+  rtctree:
+    doc:
+      type: git
+      url: https://github.com/tork-a/rtctree-release.git
+      version: release/hydro/rtctree
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/tork-a/rtctree-release.git
+      version: 3.0.1-3
+    source:
+      type: git
+      url: https://github.com/gbiggs/rtctree.git
```

This is missing a `version`, and an overall `status`:

```suggestion
      url: https://github.com/gbiggs/rtctree.git
      version: master
    status: maintained
```